messaging/src/main/java/org/axonframework/queryhandling/QueryUpdateEmitter.java

```diff
@@ -162,10 +162,24 @@
      * @param <U>              the incremental response types of the query
      * @return the object which contains updates and a registration which can be used to cancel them
      */
+    @Deprecated
```

Comment:
The JavaDoc should reference why it's deprecated. In that line of documentation, it should refer to the method/approach which should be used instead of this function.